client/src/components/DataFiles/DataFilesModals/DataFilesCompressModal.js

```diff
@@ -0,0 +1,189 @@
+import React, { useMemo } from 'react';
+import { useSelector, useDispatch, shallowEqual } from 'react-redux';
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Form,
+  Input,
+  FormGroup
+} from 'reactstrap';
+import { LoadingSpinner, FormField, Icon, InlineMessage } from '_common';
+import { useHistory, useLocation } from 'react-router-dom';
+import { isString } from 'lodash';
+import { Formik } from 'formik';
+import PropTypes from 'prop-types';
+import * as yup from 'yup';
+import './DataFilesCompressModal.module.scss';
+
+const makeDate = () => {
+  const date = new Date();
+  const d = new Date(date);
+  let month = `${d.getMonth() + 1}`;
+  let day = `${d.getDate()}`;
+  const year = d.getFullYear();
+
+  if (month.length < 2) month = `0${month}`;
+  if (day.length < 2) day = `0${day}`;
+
+  return [year, month, day].join('-');
+};
+
+const DataFilesCompressForm = ({ singleFile, formRef }) => {
+  const dateTimeStr = makeDate();
+  const initialValues = {
+    filenameDisplay: `${singleFile ? dateTimeStr : ''}`,
+    filetype: '.zip'
+  };
+  const validationSchema = yup.object().shape({
+    filenameDisplay: yup.string().required('The filename is required')
+  });
+
+  return (
+    <Formik
+      innerRef={formRef}
+      initialValues={initialValues}
+      validationSchema={validationSchema}
+    >
+      {({ values, setFieldValue }) => {
+        const handelSelectChange = e => {
+          setFieldValue('filetype', e.target.value);
+        };
+        return (
+          <Form>
+            <FormField label="Filename" name="filenameDisplay" />
+            <FormGroup>
+              <Input
+                type="select"
+                name="filetype"
+                bsSize="sm"
+                onChange={handelSelectChange}
+              >
+                <option value=".zip">zip</option>
+                <option value=".tar.gz">tar.gz</option>
```

Comment:
Please, display the extension with the dot in the options _so user can see the "final result" filename_.

```suggestion
                <option value=".zip">.zip</option>
                <option value=".tar.gz">.tar.gz</option>
```